src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -308,17 +308,20 @@ private function leave($subject, Transition $transition, Marking $marking): void
         }
     }
 
-    private function transition($subject, Transition $transition, Marking $marking): void
+    private function transition($subject, Transition $transition, Marking $marking, array $context): array
```

Comment:
Why not passing as a reference?

Reply:
I don't really know. We don't use reference too often in the Symfony world.

What do other people think about that?

Reply:
The event won't deal with a reference so I don't really see the point.